packages/xrpl/src/Wallet/index.ts

```diff
@@ -382,6 +383,34 @@ class Wallet {
     return classicAddressToXAddress(this.classicAddress, tag, isTestnet)
   }
 
+  /**
+   * Only usable on test networks.
+   * Funds an existing wallet using a faucet.
+   * This is an alias for Client.fundWallet()
+   *
+   * @param this - An existing XRPL Wallet to fund.
+   * @param client - An XRPL client.
+   * @param options - See below.
+   * @param options.faucetHost - A custom host for a faucet server. `fundWallet` will automatically
+   * determine the correct server for devnet and testnet as long as `faucetHost` is null or undefined.
+   * In other environments, you should provide the host using this option.
+   * Here's an example of how to format `faucetHost`: `faucet.devnet.rippletest.net`
+   * @returns This wallet and that its new balance in XRP.
+   * @throws When either Client isn't connected or unable to fund wallet address.
+   */
+  public async fundWallet(
```

Comment:
I don't like `fundWallet` being a wallet method. IMO the wallet should be an object that doesn't have anything to do with a client (more specifically, with a network connection) - all it does it handle the keys and do things with the keys (e.g. sign and verify). When actually using a `Wallet` in production, you don't want `fundWallet` anywhere near it.

Reply:
I agree. We should just keep `fundWallet` on `Client` and make sure the function is documented well on our docs to improve discoverability.